Urban-Fun/View Controllers/TimelineViewController.m

```diff
@@ -52,24 +61,38 @@ -(void)getActivities{
         if (activities) {
             self.arrayOfActivities = activities;
         }
-        else {
+        else if (error != nil){
+            UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Network Error"
```

Comment:
🚧 We need to refactor this into a helper class to manage the alert controller, then you can replace this call and the ones inside of SignUpViewController [like this](https://github.com/mbooker5/Urban-Fun/blob/b2b39db1ab23d4bf52748d1f660e9c9bb40f8ac3/Urban-Fun/View%20Controllers/SignUpViewController.m#L40) to use the helper class